### PR TITLE
Add routing feature

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,55 +4,75 @@ const Koa = require('koa');
...
+  if (ctx.request.body === undefined || !Array.isArray(ctx.request.body.alerts)) {
+    ctx.status = 400;
+    console.error('Unexpected request from Alertmanager:', ctx.request.body);
+    return;
+  }
+
+  const embeds = [];
+
+  ctx.request.body.alerts.forEach(alert => {
+    if (alert.annotations && (alert.annotations.summary || alert.annotations.description)) {
```
Comment (dgusakov) [inline index.js:35]: Why OR in `alert.annotations.summary || alert.annotations.description` if we use both at the end?

File: index.js [hunk 2]
```diff
@@ -4,55 +4,75 @@ const Koa = require('koa');
...
+        color: alert.status === 'resolved' ? colors.resolved : colors.firing,
       });
     }
+  });
 
-  } else {
+  if (!embeds.length) {
     ctx.status = 400;
-    console.error('Unexpected request from Alertmanager:', ctx.request.body);
+    console.warn('No data to write to embeds');
```
Comment (dgusakov) [inline index.js:46]: Make it more verbose?

File: index.js (hunk 1 above)
Comment (madlabman) [reply index.js:35]: If for some reason we omit any one of these fields it is not a problem and empty one will be just ignored. But with both summary and description empty fields notification has no sense.

File: index.js (hunk 2 above)
Comment (madlabman) [reply index.js]: Done

**Review by dgusakov — APPROVED** [review]

**Pull request merged**